### PR TITLE
Update readme to clarify what repos should be updated

### PR DESCRIPTION
**Pull request by Jag96**
@chiragsalian @tgolen will you please review this?

This PR updates the readme to clarify what repos should be updated after a new PR is merged. More context in https://expensify.slack.com/archives/C03TQ48KC/p1636044257067200.

### Fixed Issues
Discussion in [slack](https://expensify.slack.com/archives/C03TQ48KC/p1636044257067200)

# Tests/QA
N/A

### CODE REVIEW

**Review by chiragsalian — DISMISSED** [review]
LGTM

**Review by stitesExpensify — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ Alternatively, you can edit files directly in a project's `node_modules` then ap
 1. They will review and accept your changes, merge them, then deploy a new version
 
 # Deploying a Change (Expensify Only)
-1. Once the PR has been merged update the `package.json` commit hash in any repos with a dependency on `expensify-common`
+1. Once the PR has been merged update the `package.json` commit hash in any repos with a dependency on `expensify-common`. Even if your change is only required in one repo, it is important to update `package.json` in all repos that use expensify-common to prevent somebody from deploying changes they don't know about.
```
Comment (stitesExpensify) [inline README.md:29]: Do we think it's worth it to list the repos here? Someone new might not know. AFAIK it's /App, /Web-Expensify, and /Web-Secure. Any others?

Comment (Jag96) [reply README.md:29]: Mobile-Expensify is used as well, I can update this to list those

**Comment by Jag96** [comment]
@stitesExpensify updated

**Review by tgolen — APPROVED** [review]

**Review by tgolen — CHANGES_REQUESTED** [review]
Oops, need to add `Comp` repo to the list too

**Comment by Jag96** [comment]
Updated the PR to include Comp and clarify not all repos need to be updated

**Review by tgolen — APPROVED** [review]

**Review by chiragsalian — APPROVED** [review]

**Review by stitesExpensify — APPROVED** [review]

**Comment by Jag96** [comment]
Glad I don't have to update the hash for all the repos now 😂

**Pull request merged**